### PR TITLE
Dbex/63748 itf expiration fix

### PR DESCRIPTION
**Pull request by micahaspyr**
## Summary

- The formatting of the date that a veteran sees has been updated to be more clear.
- Only the way the date and time is shown has changed. The moment.js formatting was changed to 'dddd MMMM Do[,] Y [at] h[:]ma' instead of the previous 'MMMM DD, YYYY' which only showed the date of ITF expiration with no time.
- dbex team 2 aka Team Carbs

## Related issue(s)

- Link to ticket created in va.gov-team 
(https://github.com/department-of-veterans-affairs/va.gov-team/issues/63748)

## Testing done

- Previous iteration showed 'August 23, 2023' where it will now show 'Wednesday August 23rd, 2023 at 2:48pm ET'
- A user needs to log in using the 'vets.gov.user+228@gmail.com' login 
- The format of the time should look similar to the image below
Screenshot 2023-08-23 at 3 07 55 PM

- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |    
Screenshot 2023-08-23 at 3 11 31 PM
   |
| Desktop |        |    Screenshot 2023-08-23 at 3 07 55 PM   |

## What areas of the site does it impact?

This impacts the 526 form formatting of time when showing ITF expiration date and time

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [X] Linting warnings have been addressed
- [X] Screenshot of the developed feature is added
- [] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.

### Authentication

- [X] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/itfWrapper.jsx [hunk 1]
```diff
@@ -80,15 +84,15 @@ export const itfSuccess = (
...
     {hasPreviousItf && (
       <p className="vads-u-font-size--base">
         <strong>Please note:</strong> We found a previous Intent to File request
-        in our records that expired on {displayDate(prevExpirationDate)}. This
-        ITF might have been from an application you started, but didn’t finish
-        before the ITF expired. Or, it could have been from a claim you already
-        submitted.
+        in our records that expired on{' '}
+        <strong>{displayDate(prevExpirationDate)} ET</strong>. This ITF might
+        have been from an application you started, but didn’t before the ITF
```
Comment (SamStuckey) [inline src/applications/disability-benefits/all-claims/content/itfWrapper.jsx:94]: "but didn't before" sounds like a word is missing?

**Review by SamStuckey — APPROVED** [review]
LGTM

**Comment by Mottie** [comment]
I shared this with my team, since Supplemental Claims also uses the ITF endpoint. And these are the comments:

> It's breaking a few content rules:
> - Using ‘please note’ (content no longer wants to include "please" in messages)
> - ‘Additional info has to be outside of status message’
> There’s also a typo in the screenshot.

**Comment by saderagsdale** [comment]
@SamStuckey @austinjprice Can someone say more about the reason for adding a time format to include `h[:]ma`? Does the VBA have a business requirement for that, or are Veterans allowed to have the full day, regardless of the `h[:]ma` they submitted the ITF?

Asking because one of our devs wanted to know if we should make this change for the supp claims form, and I wanted clarification on the origin of the user/business requirements. Not an urgent question.

**Comment by austinjamesprice** [comment]
@saderagsdale while investigating survey and contact center feedback on ITF expiration, we encountered a number of problems with ITF expiration.  

1. ITF expiration, as calculated by the ITF end point, is exactly one year, to the hour and minute
2. Messages only included the day which could lead to users to believe they have until the end of the day
3. The VBA has different business rules for expiration that includes accepting until the end of the day plus adding additional days if the expiration lands on a holiday or weekend
4. Expiration forces users to start over with a new, blank application 

We intend to address each of these. We believe the first best step is to add time stamp to messaging to, at very least,  let Veterans know exactly when their ITF expires under the current va.gov process.

Second,  we intend to allow users to continue with an in progress application after expiration so they don't have to start over. 

Finally,  we intend to align ITF expiration with VBA rules and move expiration to end of day.  This step is more complicated and requires LH end point change, final agreement with VBA and alignment with all groups that use the ITF end point

Adding @juliepedtke for anything I missed

**Comment by Mottie** [comment]
> Expiration forces users to start over with a new, blank application

Hmmm, I never encountered this. I haven't looked at the code in a while, but I recall that an expired ITF would call the create ITF endpoint and then allow the Veteran to proceed with their saved application

**Comment by saderagsdale** [comment]
Thanks @austinjamesprice very helpful context. I'll setup time with you to learn more, don't wanna blow up the comments on this ticket.

**Review by sethdarragile6 — APPROVED** [review]

**Pull request merged**